wbooks/src/redux/book/reducer.ts

```diff
@@ -0,0 +1,30 @@
+export const initialState = {
+  books: [],
+  isLoading: false,
+  hasError: false
+};
+
+const reducer = (state = initialState, action: any) => {
+  switch (action.type) {
+    case 'GET_BOOKS':
```

Comment:
Use `actions`  constant, from `./actions` for: `'GET_BOOKS'`, `'GET_BOOKS_SUCCESS'` and `'GET_BOOKS_FAILURE'`